kinesumer.go

```diff
@@ -59,9 +60,15 @@ func NewDefaultKinesumer(awsAccessKey, awsSecretKey, awsRegion, stream string) (
 		&DefaultKinesumerOptions)
 }
 
-func NewDefaultRedisKinesumer(awsAccessKey, awsSecretKey, awsRegion, redisURL, stream string) (*Kinesumer, error) {
+func NewDefaultRedisKinesumer(awsAccessKey, awsSecretKey, awsRegion, redisURL, redisPrefix, stream string) (*Kinesumer, error) {
+	if !strings.HasPrefix(redisURL, "redis://") {
+		return nil, errors.New("Bad Redis url")
+	}
 	rss, err := NewRedisStateSync(&RedisStateSyncOptions{
-		SavePeriod: 5 * time.Second,
+		SavePeriod:  5 * time.Second,
+		AlivePeriod: 5 * time.Second,
+		RedisURL:    redisURL,
+		RedisPrefix: redisPrefix,
```

Comment:
Please add comments for each for the struct fields to indicate what they are used for (e.g. what's alive period?). Not here, but on the struct itself.